### PR TITLE
Iconography: Finances

### PR DESCRIPTION
**Pull request by Michele-Camargo**
# What

Implement the new icons proposed in the [Iconography](https://www.figma.com/file/9WVVk8Vobz2ZMzmHjtEFEh/Iconography?node-id=1825%3A1490) spec

# Why

Enable us to have a greater visual collection, once we have a greater and more diverse amount of icons

# How

Added informations components in /components/Icons

# Sample

https://user-images.githubusercontent.com/44146877/169115972-fb3fd909-4157-4860-b7d3-1734b9add9c6.mov

# QA

### CODE REVIEW

File: src/components/Icons/Finances/ChartLineUp.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function ChartBarWaterfall({
+  id = 'ChartBarWaterfallIcon',
```
Comment (lucianomlima) [inline src/components/Icons/Finances/ChartLineUp.tsx:10]: 🔧 
```suggestion
function ChartLineUp({
  id = 'ChartLineUpIcon',
```

File: src/components/Icons/Finances/ChartLineUp.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        stroke={fill}
+        strokeWidth="2"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </BaseIcon>
+  );
+}
+
+export default ChartBarWaterfall;
```
Comment (lucianomlima) [inline src/components/Icons/Finances/ChartLineUp.tsx:32]: 🔧 
```suggestion
export default ChartLineUp;
```

**Review by lucianomlima — COMMENTED** [review]
Só um pequeno detalhe xD

**Review by julianosbento — APPROVED** [review]

File: src/components/Icons/Finances/ChartColumUp.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function ChartColumUp({
+  id = 'ChartColumUpIcon',
```
Comment (albertozaranza) [inline src/components/Icons/Finances/ChartColumUp.tsx:10]: 🔧 

```suggestion
function ChartColumnUp({
  id = 'ChartColumnUpIcon',
```

And the file name

File: src/components/Icons/Finances/index.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+export { default as BankIcon } from './Bank';
+export { default as ChartBarIcon } from './ChartBar';
+export { default as ChartBarAltIcon } from './ChartBarAlt';
+export { default as ChartBarWaterfallIcon } from './ChartBarWaterfall';
+export { default as ChartCandlestickIcon } from './ChartCandlestick';
+export { default as ChartColumUpIcon } from './ChartColumUp';
```
Comment (albertozaranza) [inline src/components/Icons/Finances/index.ts:6]:
```suggestion
export { default as ChartColumnUpIcon } from './ChartColumnUp';
```

File: src/components/Icons/Finances/ChartColumUp.tsx (hunk 3 above)
Comment (lucianomlima) [reply src/components/Icons/Finances/ChartColumUp.tsx:10]: Good catch! 👌

File: src/components/Icons/Finances/ChartColumUp.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        fill={fill}
+      />
+    </BaseIcon>
+  );
+}
+
+export default ChartColumUp;
```
Comment (matheussilva2) [inline src/components/Icons/Finances/ChartColumUp.tsx:29]: Corrigir o Column aqui também

**Review by matheussilva2 — APPROVED** [review]

File: src/components/Icons/Finances/ChartLineUp.tsx (hunk 1 above)
Comment (Michele-Camargo) [reply src/components/Icons/Finances/ChartLineUp.tsx]: comments fixup in https://github.com/magnetis/astro-native/pull/582/commits/7fbac3d6421fe56175be43e4f3296e7ce53029b4

File: src/components/Icons/Finances/ChartLineUp.tsx (hunk 2 above)
Comment (Michele-Camargo) [reply src/components/Icons/Finances/ChartLineUp.tsx]: comments fixup in https://github.com/magnetis/astro-native/pull/582/commits/7fbac3d6421fe56175be43e4f3296e7ce53029b4

File: src/components/Icons/Finances/index.ts (hunk 4 above)
Comment (Michele-Camargo) [reply src/components/Icons/Finances/index.ts]: comments fixup in https://github.com/magnetis/astro-native/pull/582/commits/7fbac3d6421fe56175be43e4f3296e7ce53029b4

File: src/components/Icons/Finances/ChartColumUp.tsx (hunk 5 above)
Comment (Michele-Camargo) [reply src/components/Icons/Finances/ChartColumUp.tsx]: comments fixup in https://github.com/magnetis/astro-native/pull/582/commits/7fbac3d6421fe56175be43e4f3296e7ce53029b4

**Review by lucianomlima — APPROVED** [review]

**Pull request merged**